misc/scripts/mockgen.sh

```diff
@@ -78,7 +78,7 @@ mockgen -destination=./action/protocol/mock_protocol_test.go  \
         -package=protocol \
         Protocol
 
-mockgen -destination=./db/mock_kvstore_test.go  \
+mockgen -destination=./db/mock_kvstore.go  \
```

Comment:
modify file name, to be able to export NewMockKVStore() for use in other packages